src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Resource Hints
-description: TODO
-authors: []
-reviewers: []
-analysts: []
-editors: []
+description: Resource Hints chapter of the 2021 Web Almanac covering adoption of resource hints, their uses, bad practices and their impact on performance.
+authors: [kevinfarrugia]
+reviewers: [siakaramalegos, tunetheweb, andydavies, samarpanda]
+analysts: [nithanaroy]
+editors: [tunetheweb]
```

Comment:
```suggestion
editors: [tunetheweb, rviscomi]
```

Reply:
Happy to remove my name from editor. Can count my edits as part of my reviewer role. I just didn't want this chapter to look like it didn't have an edit if @rviscomi decided it didn't need further edits.

```suggestion
editors: [rviscomi]
```